apps/nextjs-pages-router/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@makeswift/runtime": "workspace:^",
-    "next": "14.1.0",
+    "next": "15.0.1",
```

Comment:
Upgrade `app/nextjs-pages-router` to Next.js 15, but keep it on React 18: https://nextjs.org/blog/next-15#pages-router-on-react-18

Reply:
Is there a reason not to use React 19 on this Pages Router app? Is it because it's not stable yet?

Reply:
From my reading of https://nextjs.org/blog/next-15#react-19, that seems to be a preferred community configuration. Let me know if you don't get the same read.